ingestion/src/metadata/ingestion/sink/elasticsearch.py

```diff
@@ -128,6 +130,8 @@ class ElasticSearchConfig(ConfigModel):
     timeout: int = 30
     ca_certs: Optional[str] = None
     recreate_indexes: Optional[bool] = False
+    use_AWS_credentials: Optional[bool] = False
+    region_name: str = "us-east-1"
```

Comment:
`region_name` should be an optional param

Reply:
Okay @ulixius9